app/src/main/java/com/example/keniphhomemaintenance/ui/Screen.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.keniphhomemaintenance.ui
```

Comment:
Might be worth just throwing this in `Navigation.kt`

Reply:
I think it's nice to keep it separate - I assume AndroidStudio created this file for you?

Reply:
Nah, created manually

Reply:
How did you both come up with navArgs(vararg args: String)? Did you get it from the same tutorial?

Reply:
[yes](https://www.youtube.com/watch?v=4gUeyNkGE3g&t=1s)